keps/sig-node/1539-hugepages/README.md

```diff
@@ -546,6 +560,67 @@ locality guarantees as a feature of QoS.  In particular, pods in the
     - Test case will be added to cri-tools to be used in CRI runtime' test(CI).
     - here: https://github.com/kubernetes-sigs/cri-tools
 
+## Test Plan for HugePageStorageMediumSize
+
+- Promote existing HugePages E2E tests to conformance
+
+## Production Readiness Review Questionnaire
+### Monitoring requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+An operator could audit pods that use hugepages resources.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine.
+the health of the service?**
+N/A
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+N/A
+
+* **Are there any missing metrics that would be useful to have to improve observability.
+of this feature?**
+No.
+
+### Dependencies
+
+* **Does this feature depend on any specific services running in the cluster?**
+No
+
+### Scalability
+
+* **Will enabling / using this feature result in any new API calls?**
+No.
+
+* **Will enabling / using this feature result in introducing new API types?**
+No
+
+* **Will enabling / using this feature result in any new calls to the cloud.
+provider?**
+No
+
+* **Will enabling / using this feature result in increasing size or count of.
+the existing API objects?**
+No
+
+* **Will enabling / using this feature result in increasing time taken by any.
+operations covered by [existing SLIs/SLOs]?**
+No
+
+* **Will enabling / using this feature result in non-negligible increase of.
+resource usage (CPU, RAM, disk, IO, ...) in any components?**
+No
+
+### Troubleshooting
+
+* **How does this feature react if the API server and/or etcd is unavailable?**
+No impact.
+
+* **What are other known failure modes?**
+Not applicable.
+
+* **What steps should be taken if SLOs are not being met to determine the problem?**
+Not applicable
```

Comment:
Per above, I think this is applicable. How does one troubleshoot this feature? How can one tell if it's working properly or not?

Reply:
There is no difference in troubleshooting hugepages resources with or without this feature, I believe. I thought I only need to mention what's related to HugePageStorageMediumSize. The rest is already GA, so out of scope here in my opinion.
If you think I need answer these questions for general hugepages resources I think I need help with this. I simply don't know how to answer them.

Reply:
Yes, this only pertains to HugePageStorageMediumSize. You will need to explain how cluster admins can determine if the feature is working properly for workloads and when to turn it off. It doesn't need to be more than a few sentences.